jsonschema_extractor/tests/test_attrs.py

```diff
@@ -1,25 +1,29 @@
+from typing import List
+
 import attr
 import pytest
-from attr.validators import instance_of
 from jsonschema_extractor import UnextractableSchema
 
 
-@attr.s
+@attr.define
 class Example(object):
-    integer = attr.ib(validator=instance_of(int))
-    foo = attr.ib(metadata={"jsonschema": {"type": "string", "format": "uuid"}})
-    validator_list = attr.ib(validator=[instance_of(float)])
```

Comment:
looks like this was an incorrectly labelled test, but could we keep this field since it validates the float conversion to number?

just call the field "floating_point" or something similar.

Reply:
Thank you for responding, I added the floating_point variable with a single validator